src/reducers/profile-view.js

```diff
@@ -917,15 +895,17 @@ export const selectorsForThread = (
   return selectorsForThreads[threadIndex];
 };
 
-export const selectedThreadSelectors: SelectorsForThread = (() => {
-  const anyThreadSelectors: SelectorsForThread = selectorsForThread(0);
-  const result: { [key: string]: (State) => any } = {};
+/**
+ * Create the thread selectors for the currently selected thread.
+ */
+export const selectedThreadSelectors = (() => {
+  const anyThreadSelectors = selectorsForThread(0);
+  const result: Object = {};
   for (const key in anyThreadSelectors) {
     result[key] = (state: State) =>
       selectorsForThread(UrlState.getSelectedThreadIndex(state))[key](state);
   }
-  const result2: SelectorsForThread = result;
-  return result2;
+  return (result: typeof anyThreadSelectors);
 })();
 
 export type SelectorsForNode = {
```

Comment:
Do you think you can do the same thing for this type too ?

Reply:
Yup, this would work here too.